File: Swift-C++-IR-Comparison/test/functions/Swift/StaticFunctionNoReturnMultiParameter.swift
@main
struct MyMain {

    static func main() {
        paramNoReturn(x: 42, y: 44)
    }

    static func paramNoReturn(x: Int, y: Int) {
        let a = 5
        let b = 10
        let c = a+b+x+y
    }
}

File: Swift-C++-Testsuite/test/functions/C++/StaticFunctionNoReturnMultiParameter.cpp

struct MyMain {
 
  static void paramNoReturn(int x, int y) {
    int a = 5;
    int b = 10;
    int c = a + b + x + y;
  }
};
 
int main() {
 MyMain::paramNoReturn(42, 44);
 
 return 0;
}